src/components/DayPicker.jsx

```diff
@@ -169,6 +171,7 @@ export default class DayPicker extends React.Component {
     super(props);
 
     const currentMonth = props.hidden ? moment() : props.initialVisibleMonth();
+    const translationValueRTL = getCalendarMonthWidth(props.daySize) * -1;
```

Comment:
nit: this could be `const translationValueRTL = -getCalendarMonthWidth(props.daySize);`